app/controllers/application_controller.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  include PagyBackendWithHelpers
```

Comment:
@isubas aramayı kullanmadığımız controller'lar için de `include Pagy::Backend` yapmaya gerek var mı?

Reply:
Birçok controller'da pagy kullanıyoruz, bu yüzden bu helper'ın içine koydum ve burada include ettim. Ortalama her controller içinde bir yerlerde sayfalama kullanılıyor varsayımı yaptım.

Reply:
Tamamdır. Kurul/komisyon controller'larına da bu helper'ları ekledim. Codeclimate buna bile duplicate diyor :) ignore ediyorum, FYI.